core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -8144,10 +8144,9 @@ private void checkGetTimestamp(Connection con) throws SQLException {
    * ClassCastException retrieving from ARRAY that has mixed INTEGER and DECIMAL
    * elements</a>. */
   @Test void testIntAndBigDecimalInArray() {
-    // Result should be "EXPR$0=[1, 1.1]\n"; [CALCITE-4850] logged.
     CalciteAssert.that()
         .query("select array[1, 1.1]")
-        .returns("EXPR$0=[0E+1, 1.1]\n");
```

Comment:
@tanclary thanks for patient reviewing! btw, I think we can safely close CALCITE-4850. The result has been corrected.